README.md

```diff
@@ -147,55 +122,55 @@ Each SDK will auto generate API reference documentation using the respective lan
-> Producing API reference documentation is the responsibility of an _implementer_
+> Producing API reference documentation is the responsibility of an _implementer_.
 
 ### Example Feature Usage
 
-Each SDK will **publish** example usage for _each_ implemented feature. This can either be included as a part of API reference documentation _or_ published separately
+Each SDK will **publish** example usage for _each_ implemented feature. This can either be included as a part of API reference documentation _or_ published separately.
 
 ## Test Vectors
 
 Test vectors ensure interoporability of features across SDKs and language implementations by providing common test cases with an input and expected output pair. They include both success and failure cases that can be vectorized.
 
-This repo serves as the home for all web5 feature related vectors. They are available in the [test-vectors](./test-vectors/) directory
+This repo serves as the home for all web5 feature related vectors. They are available in the [test-vectors](./test-vectors/) directory.
 
-The `tbdex` repo houses tbdex feature related vectors. They are available in the [test-vectors](https://github.com/TBD54566975/tbdex/test-vectors) directory
+The `tbdex` repo houses tbdex feature related vectors. They are available in the [test-vectors](https://github.com/TBD54566975/tbdex/test-vectors) directory.
 
-The `sdk-report-runner` repo consumes the output tests for these test vectors in each repo and generates a report - [report-runner](https://github.com/TBD54566975/sdk-report-runner)
+The `sdk-report-runner` repo consumes the output tests for these test vectors in each repo and generates a report - [report-runner](https://github.com/TBD54566975/sdk-report-runner).
 
-### Adding/Updating Vectors
+### Adding & Updating Test Vectors
 
-New test vectors should follow the standard [vector structure](./test-vectors/). Vectors are automatically validated against the JSON schema via CI.
+New test vectors should follow the standard [vector structure](./test-vectors/). Vectors are automatically validated against their [JSON Schema](https://json-schema.org/) via CI.
 
-Create a PR in this repo for adding / updating web5 test vectors
+Create a PR in this repo for adding / updating Web5 test vectors.
 
 ### Feature Completeness By SDK
 
-Test vectors are also used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
-
 ## Web5 SDK Features
 
+Test vectors are used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
+
 ### Cryptographic Digital Signature Algorithms (DSA)
 
-| Algorithm       | Typescript | Kotlin | Rust | Swift |
-| --------------- | ---------- | ------ | ---- | ----- |
-| `ES256K`        | ✅          | ✅      | ✅    | ❌     |
```

Comment:
These tables were meant to track feature support for the given implementation yeah? Seems like a good thing to track, although I agree with the removal here as this is intended to read as a spec doc (the word "spec" being in the repo and all). @frankhinek any opinions on this matter? Cool with just axing the tracking altogether at this time? Seems like this is appropriate per-implementation repo, but may be good to be consistent in our approach.

Reply:
above i reference https://tbd54566975.github.io/sdk-report-runner/ which should be our source of truth (dynamically generated)